app/validators/blocks/relationship_collector_validator.py

```diff
@@ -1,5 +1,6 @@
 from app import error_messages
 from app.validators.blocks.block_validator import BlockValidator
+from app.validators.blocks.unrelated_block_validator import UnrelatedBlockValidator
```

Comment:
A minor niggle, but these imports could be:

```
from app.validators import BlockValidator
from app.validators import UnrelatedBlockValidator
```

with some small changes to the init modules

Reply:
I agree, but this is consistent with the imports throughout validator. This change should be made across all files in a separate pull request.